### ISSUE TITLE
Incomplete nodes loading

### ISSUE THREAD

**Issue by scotmerrick** [BME-nodeservers/docs#7]
Just added the ambient weather node to my Polisy (PG3, IoX 5.6.2) and successfully added my api.  It shows up on the ISY, and has 3 nodes.  Main and indoor only show pressure and temp, so I am missing a bunch of variables in the "main" node.  I have tried rebooting ISY.  Here is the Polisy log for the server
[AmbientWeather_6-27-2023_104708-AM.zip](https://github.com/BME-nodeservers/docs/files/11886766/AmbientWeather_6-27-2023_104708-AM.zip)

**Comment by bpaauwe** [comment]
Looks like you have your log level still at error level so it would only shows errors.  It's not showing any errors.

Setting the log level to debug and letting it run for 5 or 10 minutes, download that log and attach it and I might be able to determine what's going on.

**Comment by scotmerrick** [comment]
Thank you.  Here is the log.  So far, I have deleted and re-installed the node server and I entered a new api from the Ambient website ( as I saw that the first api was not accepted).  Now, things more promising. On  UD mobile all the nodes are populated.  But on IoX, not much.  I am attaching screen shots of what I see on IoX.

 

Scot Merrick

 

From: Bob Paauwe ***@***.***>
Reply-To: BME-nodeservers/docs ***@***.***>
Date: Tuesday, June 27, 2023 at 2:46 PM
To: BME-nodeservers/docs ***@***.***>
Cc: Merrick Mill Valley ***@***.***>, Author ***@***.***>
Subject: Re: [BME-nodeservers/docs] Incomplete nodes loading (Issue #7)

 

Looks like you have your log level still at error level so it would only shows errors. It's not showing any errors.

Setting the log level to debug and letting it run for 5 or 10 minutes, download that log and attach it and I might be able to determine what's going on.

**Comment by bpaauwe** [comment]
Hi Scott,

Nothing got attached to the github comment. Maybe you can't add attachments via email responses?

In any case, if things are looking proper in UD Mobile then it's not likely to be the node server.  Try restarting the admin console and see if that makes show everything correctly.

**Comment by scotmerrick** [comment]
Here is a new log and the screen shots from AC.  I tried rebooting the AC but no difference.
[AmbientWeather_6-28-2023_92625-AM.zip](https://github.com/BME-nodeservers/docs/files/11898954/AmbientWeather_6-28-2023_92625-AM.zip)
<img width="1124" alt="Screen Shot 2023-06-27 at 6 35 28 PM" src="https://github.com/BME-nodeservers/docs/assets/47925246/bd5b21d8-78e2-4905-adbc-229e59253618">
<img width="1124" alt="Screen Shot 2023-06-27 at 6 35 16 PM" src="https://github.com/BME-nodeservers/docs/assets/47925246/53778f37-7cc1-4391-a7ad-13682660cd9a">

**Comment by bpaauwe** [comment]
Everything in the log looks correct.  The node server looks to be sending the right values for the Main node.  

In PG3, from the dashboard, select the Ambient node server.  You should see a button called "Load Profile".  Click that.  Then restart the admin console.  See if that corrects it.

If not, do you know how to view the Java console?  If you're on Windows, I believe you can enable it via the Java app in the control panel. If you enable that and then start the admin console, it may show some errors related to parsing the profile files.  I'd need to see what those errors are.

**Comment by scotmerrick** [comment]
OK.  I did the Profile then reboot, with no changes seen.  I am on a mac so I opened the Java control panel and enabled tracing and logging.  
The attached screen shots are all that I can find in the control panel.
<img width="872" alt="Screen Shot 2023-06-28 at 1 04 25 PM" src="https://github.com/BME-nodeservers/docs/assets/47925246/7997959a-074e-4540-ac69-91751c6b3a0d">
<img width="1014" alt="Screen Shot 2023-06-28 at 1 03 12 PM" src="https://github.com/BME-nodeservers/docs/assets/47925246/659b1ac3-4668-4a2a-bebe-7df7577a81a8">

**Comment by scotmerrick** [comment]
Did you receive the Java logs?Scot Merrick On Jun 28, 2023, at 12:10 PM, Bob Paauwe ***@***.***> wrote:﻿
Everything in the log looks correct.  The node server looks to be sending the right values for the Main node.
In PG3, from the dashboard, select the Ambient node server.  You should see a button called "Load Profile".  Click that.  Then restart the admin console.  See if that corrects it.
If not, do you know how to view the Java console?  If you're on Windows, I believe you can enable it via the Java app in the control panel. If you enable that and then start the admin console, it may show some errors related to parsing the profile files.  I'd need to see what those errors are.

**Comment by bpaauwe** [comment]
No, just a couple of screen shots that don't help any.  I don't know how to enable the Java console logging in a Mac.

**Comment by scotmerrick** [comment]
So I upgraded to PG3x from PG3 to see if that made a difference. First try no different so I deleted the node server , re-installed it and rebooted polisy and Iox. Now, the nodes are not showing at all in ISY and UD mobile gives me the attached error messageSHMOn Jul 1, 2023, at 10:05 AM, Bob Paauwe ***@***.***> wrote:﻿
No, just a couple of screen shots that don't help any.  I don't know how to enable the Java console logging in a Mac.

**Comment by scotmerrick** [comment]
Problem solved. Somehow the ISY address in Policy changed from “local host” to an IP numberSHMOn Jul 5, 2023, at 5:22 PM, Scot Merrick ***@***.***> wrote:﻿So I upgraded to PG3x from PG3 to see if that made a difference. First try no different so I deleted the node server , re-installed it and rebooted polisy and Iox. Now, the nodes are not showing at all in ISY and UD mobile gives me the attached error message<IMG_2006.png>SHMOn Jul 1, 2023, at 10:05 AM, Bob Paauwe ***@***.***> wrote:﻿
No, just a couple of screen shots that don't help any.  I don't know how to enable the Java console logging in a Mac.
